src/hackney_http.erl

```diff
@@ -293,8 +293,10 @@ parse_header(Line, St) ->
                  end,
   St1 = case hackney_bstr:to_lower(hackney_bstr:trim(Key)) of
           <<"content-length">> ->
-            CLen = list_to_integer(binary_to_list(hackney_bstr:trim(Value))),
-            St#hparser{clen=CLen};
+            case catch list_to_integer(binary_to_list(Value)) of
```

Comment:
any reason to not use `binary_to_integer/1` ? though not sure which version of erlang it appeared ...